inst/rmarkdown/templates/pdf/resources/template.tex

```diff
@@ -189,7 +189,11 @@
 \fi
 $endif$
 
-\usepackage{graphicx,grffile}
+\usepackage{graphicx}
+% grffile has become a legacy package: https://ctan.org/pkg/grffile
+\IfFileExists{grffile.sty}{%
```

Comment:
This test doesn't really do any harm but (other than for a few hours due to a bug in the texlive distribution over the weekend) it will always be true as we still distribute grffile, it just does nothing other than include graphicx, specifically so the first version would still work without requiring any change.

Reply:
Thanks for the assessment @davidcarlisle. 

Do you suggest the problem has already passed? Or might the grffile still be missing for sometime in Tex Live distributions of various Linux distributions due to time lags? In that case the test might be helpful for a certain time. But if the probability of missing grffile is very low it might be more sensible to take the test out again now.

Reply:
that problem (that it was not in the texlive server at all) will have passed, it was only missing for a few hours so unless sites were regularly updating they would never have seen the issue, and if they have updated since, the issue has gone... however as grffile doesn't do anything now texlive may move it out of the latex-recommended collection to latex-extra so people installing a more minimal texlive may not have  it installed locally (this possibility hadn't occurred to me when I made the original comment)